cpp/src/cider/type/plan/BinaryExpr.cpp

```diff
@@ -258,14 +258,16 @@ JITExprValue& BinOper::codegenFixedSizeLogicalFun(CodegenContext& context,
   if (branchless_logic) {
     switch (get_optype()) {
       case kAND: {
-        // If one side is not null and false, return not null and FALSE
-        // else no special handle needed
+        //|     |TRUE |FALSE|NULL |
+        //|TRUE |TRUE |FALSE|NULL |
+        //|FALSE|FALSE|FALSE|FALSE|
```

Comment:
you can use 0, 1 in validate position: 0: null, 1: not null

Reply:
Maybe use this single table to show both validity results and value results?